docs/training/2025/webinar-series-2025Q2.md

```diff
@@ -48,51 +54,101 @@ Attendance is free of cost, but registration is required.
+in an efficient way.
+
+In this webinar, we will introduce you to CernVM-FS, show how to access existing CernVM-FS repositories (like [EESSI](https://eessi.io),
+cover some aspects specific to using CernVM-FS on HPC systems.
+
+It is intended for people with a background in HPC (system administrators, support team members, end users, etc.) who are new to CernVM-FS:
+no specific prior knowledge or experience with it is required.
 
 ---
 
-### Introduction to EasyBuild (19 May 2025)
+### Introduction to EasyBuild
 
-- Monday 19 May 2025 (13:30-15:30 CEST)
+- Monday 19 May 2025, 13:30-15:30 CEST
 - speakers:
-    - _more info soon_
-- outline
-    - incl. highlighted features in EasyBuild 5.0.0
+    - Kenneth Hoste (Ghent University)
 
-### EESSI for CI/CD (26 May 2025)
+#### Outline
 
-- Monday 26 May 2025 (13:30-15:30 CEST)
+EasyBuild (https://easybuild.io) is a open-source software installation tool written in Python that aims to support
+the various installation procedures used by the vast collection of software packages that are typically installed from source code
+in an HPC environment.
+
+In this webinar, we will introduce you to EasyBuild, show to install and configure it, and present basic usage through
+hands-on demos. We will also cover some new capabilities that are supported by the recently released EasyBuild v5.0.0.
+
+---
+
+### EESSI for CI/CD
+
+- Monday 26 May 2025, 13:30-15:30 CEST
 - speakers:
-    - _more info soon_
-- outline
-    - _more info soon_
+    - Alan O'Cais (CECAM, University of Barcelona)
+
+#### Outline
+
+*(more info soon)*
 
 ---
 
-### Using EESSI as the base for a system stack (2 June 2025)
+### Using EESSI as the base for a system stack
 
-- Monday 2 June 2025 (13:30-15:30 CEST)
+- Monday 2 June 2025, 13:30-15:30 CEST
 - speakers:
-    - _more info soon_
-- outline
-    - _more info soon_
+    - Bob Dröge (University of Groningen)
+    - Pedro Santos Neves (University of Groningen)
+
+#### Outline
+
+Despite being very open to community contributions, EESSI is unlikely to provide all the software you want to offer
+on your system as an HPC site. For instance, it will never be able to provide proprietary software due to license constraints.
+
+This session focuses on how you can use EESSI as the base for a system software stack, and easily build additional software on top
+and add it to your local software share. A recommended setup using CernVM-FS will be shown, but pointers for using any other (shared)
+filesystem will also be given. Finally, we will show how the EESSI build workflow (using a build bot and pull requests) can be adopted
+for your local software installations.
+
+This webinar is particularly intended for system administrators or support team members that maintain the central software stack
+for the users of their HPC infrastructure, but it can also be useful for end users who want to install additional software.
+
+---
+
+!!! tip
```

Comment:
Is it intentional that you give this tip twice?

Reply:
Intentional or not, I'd keep it here too :wink:

Reply:
Yes, bottom and top of the page.